tests/Doctrine/Performance/Mock/NonProxyLoadingEntityManager.php

```diff
@@ -126,15 +128,15 @@ public function rollback() : void
     /**
      * {@inheritDoc}
      */
-    public function createQuery($dql = '') : Query
+    public function createQuery(string $dql = '') : Query
```

Comment:
While introducing BC breaks for the next major release, should we keep the default empty string? Does it make sense to have that? Or is this part of a later discussion?

Reply:
Adding parameter type is not a BC break. 
At this time I don't want to change logic or accepted types too much where not needed, otherwise the PR(s) would be pain to review.

Reply:
Oh, right. Type widening in PHP 7.2. PHP 7.2 will be minimum version. I totally forgot that.